search/service_others.go

```diff
@@ -0,0 +1,7 @@
+// +build !windows
```

Comment:
Please use `github.com/IMQS/gowinsvc/service` to make this a service. You can call `if !service.RunAsService` in main.go instead of using these